Components/Combat/DefaultTargetingProvider.cs

```diff
@@ -251,13 +251,17 @@ public bool IsInRange(Vector3 position, TrinityPower power)
                 }
             }
 
+            // Some calculated values are 3.8 and the minimum possible destination is 4.0 to come by the object.
+            // However the real interactable range is 7.0f.
+            rangeRequired = Math.Max(7f, rangeRequired);
+
             Core.Logger.Verbose(LogCategory.Targetting, $">> CurrentPower={power} CurrentTarget={position} RangeReq:{rangeRequired} Dist:{distance}");
 
             // Handle Belial differently, he's never in LineOfSight.
             if (Core.Player.IsInBossEncounter && currentTarget != null && currentTarget.ActorSnoId == (int) SNOActor.Belial)
                 return distance <= rangeRequired;
 
-            return distance <= rangeRequired && IsInLineOfSight(position);
+            return distance <= rangeRequired && (distance < 7f || IsInLineOfSight(position));

```

Comment:
`distance < 7f` check is redundant as it will be at worst 7.00 due to assignment on line 256

Reply:
It is not redundant but the logic is wrong. It should be <= 7.0f. The thing is, actors with a very small distance are failing IsLineOfSight which causes that interaction does not work.